test/functional/specs/Audiences/C12412.js

```diff
@@ -65,5 +65,8 @@ test(`Verify cookie is set on the / path `, async () => {
   await alloy.sendEvent();
 
   const cookies = await t.getCookies("C12412");
-  await t.expect(cookies[0].path).eql("/");
+
+  if (cookies.length > 0) {
```

Comment:
Can you add a comment here to explain that Firefox cannot read the cookies path? Also, does Firefox just return an empty object?

Reply:
Done. In Firefox an empty array is returned even if the cookie exists.